doc/python/axes.md

```diff
@@ -380,6 +380,30 @@ fig.update_layout(title_text="Apple Stock Price")
 fig.show()
 ```
 
+#### Adding minor ticks
+
+_new in 5.8_
+
+You can add minor ticks to an axis with `minor`. This takes a `dict` of properties to apply to minor ticks. 
+
```

Comment:
It includes the following properties: 
`tickmode`, `tickvals`, `tickcolor`, `ticklen`, `tickwidth`, `dtick`, `tick0`, `nticks`, `ticks`,
`showgrid`, `gridcolor`, `griddash` and `gridwidth`.